SimpleApp/ViewController.swift

```diff
@@ -2,34 +2,148 @@ import UIKit
 
 class ViewController: UIViewController {
     @IBOutlet private(set) var label: UILabel!
+    @IBOutlet weak var usernameTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var button: UIButton!
+    @IBOutlet weak var saveCredsSwitch: UISwitch!
+    @IBOutlet weak var savePasswordLabel: UILabel!
+
+    private enum Constants {
+        static let userNameKey = "userNameKey"
+        static let passwordKey = "passwordKey"
+        static let isCheckedKey = "isCheckedKey"
+        static let username = "user"
+        static let password = "password"
+    }
+
+    private var username: String?
+    private var password: String?
+    private var isChecked = false
+
+    @IBAction func saveCredsSwitchChanged() {
+        removeNamePasswordIfNeeded()
+    }
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        usernameTextField.delegate = self
+        passwordTextField.delegate = self
+        passwordTextField.isSecureTextEntry = false
```

Comment:
`isSecureTextEntry = false` is default value, assume, `true` is desired behavior

Reply:
fixed